Tests for Add and Delete functions

sumairq opened the pull request:
# Testing Add and Delete functions
his pull request is about testing the add and delete functions of the to-do list project, using Jest

### Added the following 

- Mocks for local storage
- Test scenarios for Add and Delete 
- ES6 import-export syntax
- JSDOM environment

### Files created for this pull request:

- __mocks__/getLocalStorage.js
- tests/addRemove.test.js
- src/getLocalStorage.js

sumairq committed:
Add es6 support in jest

sumairq committed:
Cleaned up to do class

sumairq committed:
Create tests folder

sergiogval committed:
Refactor to create pure functions

sumairq committed:
Add tests for create item

sumairq committed:
Add test for delete function

sumairq committed:
Fix linter errors

elisha2kyakpo1 reviewed (changes requested):
Hi team,

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight

- I like the way you have made the PR look very professional(descriptive title and PR description).✔✔
- All the tests are passing, keep it up✔✔

## Required Changes ♻️
_Check the comments under the review._

- [x] Let's update the readme file to include the test command instructions, this is because we are testing in this milestone. It's important that we let users know how they can test the project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sergiogval committed:
Update readme

sergiogval committed:
Update README.md

I included my info as an author of the project, and did some changes regarding the links for the demo live version as well as for clone the project. Let me know what you think.

sumairq committed:
Update readme file to include instructions about testing

ShoiraTa commented:
### **STATUS: APPROVED  ✔️** 
------
Hi team,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
